docs/data/migration/migration-data-grid-v5/migration-data-grid-v5.md

```diff
@@ -60,12 +60,23 @@ Below are described the steps you need to make to migrate from v5 to v6.
 
 - The `GridColDef['hide']` property was removed. Use `GridColDef['columnVisibility']` instead.
 
+### Rows
+
+- The `GridRowParams['getValue']` property was removed. Use `params.row` instead.
+- The `GridCellParams['getValue']` property was removed. Use `params.row` instead.
+- The `GridActionsCellProps['api']` property was removed. Use `useGridApiContext` hook instead to get `apiRef`.
+- The `GridActionsCellProps['getValue']` property was removed. Use `params.row` instead.
+- The `GridFooterCellProps['getValue']` property was removed. Use `params.row` instead.
+
 ### `apiRef` methods
 
+- The `apiRef.current.updateColumn` method was removed. Use `apiRef.current.updateColumns` instead.
+- The `apiRef.current.getColumnsMeta` method was removed. Use `gridColumnsTotalWidthSelector` or `gridColumnPositionsSelector` selectors instead.
+- The `apiRef.current.getRowIndex` method was removed. Use `apiRef.current.getRowIndexRelativeToVisibleRows` instead.
 - Some internal undocumented `apiRef` methods and properties were removed.
 
   If you don't use undocumented properties - you can skip the list below.
-  Otherwise, please check the list below and [let us know](https://github.com/mui/mui-x/issues/new/choose) if there's something missing in the `apiRef`.
+  Otherwise, please check the list and [let us know](https://github.com/mui/mui-x/issues/new/choose) if there's something missing in the `apiRef`.
```

Comment:
```suggestion
  Otherwise, please check the list and [open an issue](https://github.com/mui/mui-x/issues/new/choose) if there's something missing in the `apiRef`.
```